assets/src/components/button/index.js

```diff
@@ -17,7 +22,8 @@ class Button extends Component {
 	 */
 	render( props ) {
 		const { value } = this.props;
-		return <BaseComponent className="muriel-button" { ...this.props } />;
+		const { classNames } = this.props;
+		return <BaseComponent { ...this.props } className={ murielClassNames( 'muriel-button', this.props.className ) }  />;
```

Comment:
Should `this.props.className` here actually be `this.props.classNames` (just the one letter difference)?